.travis.yml

```diff
@@ -1,8 +1,8 @@
 ---
 language: "node_js"
-node_js: [ "4", "5", "6", "7", "8", "9", "10", "11", "node" ]
+node_js: [ "8", "9", "10", "11", "node" ]
```

Comment:
Not 12 and 13?
8 is already EOL btw.

Reply:
That's from a while ago. I'm not sure 12 was LTS at the time.

Reply:
Ah yeah, this PR has been around for a bit :)

Reply:
Every version of Node.js currently supported by the Node project ought to be supported by `licensee`. Do you happen to know offhand what those versions are today?

I'm actually disappointed to drop support for any Node.js versions at all. The APIs `licensee` needs have been around, and stable, since very early on. In that sense there's no technical reason to drop support. But `licensee` relies on dependencies, including `read-package-tree` and `semver` from the npm CLI itself, that have adopted newly supported syntax, and occasionally new APIs.